src/include/network/network_types.h

```diff
@@ -41,5 +41,14 @@ enum class Transition {
   NEED_SSL_HANDSHAKE,
   NEED_WRITE
 };
+
+enum class ResponseProtocol {
+  // No response required (for intermediate messgaes such as parse, bind, etc.)
```

Comment:
Parse and bind would require responses(ParseOK, BindOK etc). They just do not need responses immediately.

Reply:
I think this is a legacy from a previous refactor that is no longer used. Will remove.